crates/router/src/connector/cybersource.rs

```diff
@@ -633,10 +633,17 @@ impl ConnectorIntegration<api::RSync, types::RefundsData, types::RefundsResponse
         req: &types::RefundSyncRouterData,
         connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
+        let refund_id = req
+            .response
```

Comment:
as mentioned in #445 can you instead take the refund id from [RefundsData](https://github.com/juspay/hyperswitch/blob/main/crates/router/src/types.rs#L33) since we should not be using response field to construct the request.